quiz.js

```diff
@@ -37,9 +39,8 @@ db.dropDatabase();
 
 // your code begins here
 
-
-
-
+db.westeros.update({house: 'Stark'}, {$push: {people: {name: 'Arya Stark'}}});
```

Comment:
This will add one person - what if we wanted to add multiple people? We could either write a separate $push statement  for each person, or we could use Mongo's $each operator, as follows:
```
db.westeros.update({ house : 'Stark' '}, { $push : { people : 
  { $each : [
    { name : 'Ned Stark' },
    { name : 'Arya Stark' },
    { name : 'Sansa Stark' }
  ] }
} })
```